packages/react-native/src/primitives/Label/styles.ts

```diff
@@ -1,7 +1,47 @@
 import { StyleSheet } from 'react-native';
 
-import { LabelStyles } from './types';
+import { StrictTheme } from '../../theme';
+import { LabelStyles, FontWeightType } from './types';
 
-export const styles: LabelStyles = StyleSheet.create({
-  label: { fontSize: 16, marginVertical: 2, marginHorizontal: 4 },
-});
+export const getThemedStyles = (theme: StrictTheme): LabelStyles => {
+  const { colors, fontSizes, fontWeights, space, components } = theme.tokens;
+
+  return StyleSheet.create({
+    text: {
+      color: colors.font.primary,
+      fontSize: fontSizes.medium,
+      fontWeight: fontWeights.normal as FontWeightType,
```

Comment:
Do we need to cast this?

Reply:
Yes, because RN doesn't accept string for this property.

Reply:
I wouldn't expect that.